src/Book.js

```diff
@@ -0,0 +1,72 @@
+import React, {useState, useEffect} from 'react';
+import './App.css';
+import {loadJson} from './sendJson';
+import {useParams} from 'react-router-dom';
+
+const loadOneBook = async ({id}, onLoad, setError) => {
+  const result = await loadJson('/getOneBook/' + id);
+  if (result.error) {
+    setError(result.error);
+  } else {
+    onLoad(result);
+  }
+};
+
+export const Book = () => {
+  const [bookState, setBookState] = useState([]);
+  const [error, setError] = useState('');
+  const id = useParams();
+
+  useEffect(() => {
+    loadOneBook(id, setBookState, setError);
+  }, []);
+
+  return (
+    <div className="container">
+      <div className="container-fluid">
+        <button type="button" className="btn btn-outline-success btn-lg">
```

Comment:
These three buttons don't do anything, right? Can they be deleted?

Reply:
https://martinfowler.com/bliki/Yagni.html

Reply:
Great read. Makes a lot of sense. I will remove these buttons.